SignalMessaging/Info.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
FYI the app id I selected for the share extension is: `org.whispersystems.signal.shareextension` and the app group id is: `group.org.whispersystems.signal.group`.  

It would have been more standard and aesthetically pleasing to use `group.org.whispersystems.signal` but that was already taken, I'm guessing by an OWS dev before our time.